modules/nw-networkpolicy-about.adoc

```diff
@@ -123,3 +123,29 @@ spec:
 `NetworkPolicy` objects are additive, which means you can combine multiple `NetworkPolicy` objects together to satisfy complex network requirements.
 
 For example, for the `NetworkPolicy` objects defined in previous samples, you can define both `allow-same-namespace` and `allow-http-and-https` policies within the same project. Thus allowing the pods with the label `role=frontend`, to accept any connection allowed by each policy. That is, connections on any port from pods in the same namespace, and connections on ports `80` and `443` from pods in any namespace.
+
+[id="nw-networkpolicy-allow-from-router_{_content-type}"]
+== allow from router
+
+Allows router regardless of the configuration:
```

Comment:
Allows traffic from router regardless of endpoint publishing strategy.

Reply:
Can also add allow-from-hostnetwork policy
<pre>
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: allow-from-hostnetwork
spec:
  ingress:
  - from:
    - namespaceSelector:
        matchLabels:
          policy-group.network.openshift.io/host-network: ""
  podSelector: {}
  policyTypes:
  - Ingress
  </pre>

Reply:
As another part or step?